drivers/input/input_cst816s.c

```diff
@@ -14,7 +14,8 @@
 
 LOG_MODULE_REGISTER(cst816s, CONFIG_INPUT_LOG_LEVEL);
 
-#define CST816S_CHIP_ID 0xB4
+#define CST816S_CHIP_ID1 0xB4
+#define CST816S_CHIP_ID2 0xB5
```

Comment:
may be worthwile adding a short comment before these mentioning that the first comes from the official documentation and the second has been found on some devices but is not documented

Reply:
If you are able to point me the official doc, good, but even that I'm not able to find it.